common/test/acceptance/tests/lms/test_lms_gating.py

```diff
@@ -143,15 +156,51 @@ def test_gated_subsection_in_lms(self):
         self._setup_prereq()
         self._setup_gated_subsection()
 
-        self._auto_auth(self.USERNAME, self.EMAIL, False)
+        self._auto_auth(self.STUDENT_USERNAME, self.STUDENT_EMAIL, False)
 
         self.courseware_page.visit()
         self.assertEqual(self.courseware_page.num_subsections, 1)
 
         # Fulfill prerequisite and verify that gated subsection is shown
-        problem_page = ProblemPage(self.browser)
-        self.assertEqual(problem_page.wait_for_page().problem_name, 'HEIGHT OF EIFFEL TOWER')
-        problem_page.click_choice('choice_1')
-        problem_page.click_check()
+        self._fulfill_prereqruisite()
+        self.courseware_page.visit()
+        self.assertEqual(self.courseware_page.num_subsections, 2)
+
+    def test_gated_subsection_in_lms_for_staff(self):
+        """
+        Given that I am a staff member
+        When I visit the LMS Courseware
+        Then I can see all gated subsections
+        Displayed along with notification banners
+        Then if I masquerade as a student
+        Then I cannot see a gated subsection
+        When I fufill the gating prerequisite
+        Then I can see the gated subsection (without a banner)
+        """
+        self._setup_prereq()
+        self._setup_gated_subsection()
+
+        # Fulfill prerequisites for specific student
+        self._auto_auth(self.STUDENT_USERNAME, self.STUDENT_EMAIL, False)
+        self.courseware_page.visit()
+        self._fulfill_prereqruisite()
+
+        self._auto_auth(self.STAFF_USERNAME, self.STAFF_EMAIL, True)
+
         self.courseware_page.visit()
+        staff_page = StaffPage(self.browser, self.course_id)
+        self.assertEqual(staff_page.staff_view_mode, 'Staff')
+        self.assertEqual(self.courseware_page.num_subsections, 2)
+        self.q(css=self.subsection_selector)[1].click()
```

Comment:
`q` is a method on the page object, so you'll want `self.staff_page.q...` (assuming `staff_page` is the correct page)